glad/documentation.py

```diff
@@ -0,0 +1,190 @@
+import re
+import glad.util
+from lxml import etree
```

Comment:
Does this work without lxml as well? Ideally lxml is optional and just a performance improvement but not necessary.

Reply:
Done. Now it works with both lxml and xml